packages/wallet-sdk/src/core/message/RPCMessage.ts

```diff
@@ -15,9 +15,10 @@ export type EncryptedData = {
 };
 
 export interface RPCRequestMessage extends RPCMessage {
+  callbackUrl?: string;
```

Comment:
dude. you gotta self review your PR before asking another approval @nateReiners

Reply:
[6ab13a0](https://github.com/coinbase/coinbase-wallet-sdk/pull/1392/commits/6ab13a09fb821d8df468c41d6019886c17e9825c) 

My bad, tried to move to fast! Looks good to me now 🙏